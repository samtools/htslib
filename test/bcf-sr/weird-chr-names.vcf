##fileformat=VCFv4.3
##reference=ref.fa
##contig=<ID=1>
##contig=<ID=1:1>
##contig=<ID=1:1-1>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	1	.	C	T	.	.	.
1	2	.	C	T	.	.	.
1:1	1	.	C	T	.	.	.
1:1	2	.	C	T	.	.	.
1:1-1	1	.	C	T	.	.	.
1:1-1	2	.	C	T	.	.	.
