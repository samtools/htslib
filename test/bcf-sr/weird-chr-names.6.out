##fileformat=VCFv4.3
##FILTER=<ID=PASS,Description="All filters passed">
##reference=ref.fa
##contig=<ID=1>
##contig=<ID=1:1>
##contig=<ID=1:1-1>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1:1-1	1	.	C	T	.	.	.
